src/main/scala/com/exasol/cloudetl/kinesis/KinesisUserProperties.scala

```diff
@@ -65,6 +70,32 @@ class KinesisUserProperties(val propertiesMap: Map[String, String])
   final def getTableName(): String =
     getString(TABLE_NAME_PROPERTY)
 
+  final def containsMaxRecordsPerRun(): Boolean =
+    containsKey(MAX_RECORDS_PER_RUN_PROPERTY)
+
+  final def getMaxRecordsPerRun(): String =
+    getString(MAX_RECORDS_PER_RUN_PROPERTY)
```

Comment:
Use `get` with default value, and return a number. 

The caller then can use if else/pattern-matching and return the value in the importer.

Some examples [here](https://github.com/exasol/cloud-storage-etl-udfs/blob/master/src/main/scala/com/exasol/cloudetl/kafka/KafkaConsumerProperties.scala#L54-L62).

Reply:
I'm not sure that I need a default value. Kinesis already has a default value set to 10 000 inside itself for this property.

Reply:
`Limit
The maximum number of records to return. Specify a value of up to 10,000. If you specify a value that is greater than 10,000, GetRecords throws InvalidArgumentException. The default value is 10,000.`

Reply:
:+1: